openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java

```diff
@@ -1827,6 +1828,102 @@ default Class<IngestionPipeline> getEntityClass() {
     default String getNameColumn() {
       return "fullyQualifiedName";
     }
+
+    @Override
+    default int listCount(ListFilter filter) {
```

Comment:
we can move this to CollectionDAO, listCount , list after can have a different behaviour based on the query param, check for queryEntitiy what we did with "entityId"